convoy/convoy.py

```diff
@@ -24,4 +25,6 @@ def __init__(self, config):
         self.endpoint = endpoint.Endpoint(config)
         self.event_delivery = event_delivery.EventDelivery(config)
         self.event = event.Event(config)
-        self.group = group.Group(config)
\ No newline at end of file
+        self.group = group.Group(config)
+        self.sources = source.Source(config)
```

Comment:
Do we want to use singular or plural forms here? The other resources currently use the singular form.

Reply:
hm. I think i'll just use the singular then.